From a6daaf8efd102702d1ebbde21a049790a4b7d9d6 Mon Sep 17 00:00:00 2001
Message-ID: <a6daaf8efd102702d1ebbde21a049790a4b7d9d6.1740173176.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1740173176.git.sharlatanus@gmail.com>
References: <cover.1740173176.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 14 Feb 2025 20:22:53 +0000
Subject: [PATCH 07/51] gnu: python-asdf: Update to 4.1.0.

* gnu/packages/astronomy.scm (python-asdf): Update to 4.1.0.
[arguments] <phases>: Remove 'patch-pyproject-toml.
[native-inputs]: Remove python-setuptools-scm; add
python-setuptools-scm-next.

Change-Id: Ice7068e715146c5cd49eca402d3f3434ab26ede7
---
 gnu/packages/astronomy.scm | 16 ++++------------
 1 file changed, 4 insertions(+), 12 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 9af0bad5edc..a91bc5e523c 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1663,26 +1663,18 @@ (define-public python-aplpy
 (define-public python-asdf
   (package
     (name "python-asdf")
-    (version "4.0.0")
+    (version "4.1.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "asdf" version))
        (sha256
-        (base32 "1nzrgsyidcynhgiqxy7pwny07prza1g6b7hs17ybg6fh9zas6kc6"))))
+        (base32 "189d282d8c3yyv5fk473rvfm5yra05zsn4lmkaynimszr294kx0g"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
-      #~(list "--numprocesses" (number->string (parallel-job-count)))
-      #:phases
-      #~(modify-phases %standard-phases
-          (add-after 'unpack 'patch-pypojrect-toml
-            (lambda _
-              (substitute* "pyproject.toml"
-                ;; TypeError: Configuration.__init__() got an unexpected
-                ;; keyword argument 'version_file'
-                (("version_file = \"asdf/_version.py\"") "")))))))
+      #~(list "--numprocesses" (number->string (parallel-job-count)))))
     (native-inputs
      (list python-fsspec
            python-psutil
@@ -1690,7 +1682,7 @@ (define-public python-asdf
            python-pytest-doctestplus
            python-pytest-remotedata
            python-pytest-xdist
-           python-setuptools-scm
+           python-setuptools-scm-next
            python-wheel))
     (propagated-inputs
      (list python-asdf-standard
-- 
2.47.1

